bin/analyze_hoc_activity

```diff
@@ -172,15 +175,14 @@ def main()
     lines_of_code:            lines_of_code, #JSON.parse(Net::HTTP.get(URI('http://archive.learn.code.org/line_data.json')))['all'],
   }
 
-  total_females = DASHBOARD_DB[:users].where(gender: 'f').count;
-  total_males = DASHBOARD_DB[:users].where(gender: 'm').count;
-  total_with_gender = total_females + total_males
-  total_with_gender = 1 if total_with_gender == 0
+  total_females = DASHBOARD_DB[:users].where(gender: 'f').where.not(last_sign_in_at: nil).count;
+  total_males = DASHBOARD_DB[:users].where(gender: 'm').where.not(last_sign_in_at: nil).count;
```

Comment:
Don't they have to have signed in at least once in order for us to have gender information?